src/ontology/mondo-edit.obo

```diff
@@ -574661,6 +574663,16 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1040031
+name: dyneinopathy
+def: "Spectrum of disease related to monoallelic variants in DYNC1H1 and characterized by variable neuromuscular and/or neurodevelopmental presentations. While not absolute, there appear to be genotype-phenotype correlations based on the location of the variant. Patients with variants in the stem domain of DYNC1H1 have been reported with a predominantly neuromuscular presentation, including congenital myopathy, spinal muscular atrophy, Charcot-Marie-Tooth (CMT), and less frequently, intellectual disability and autism. Patients with variants in the motor domain predominantly present with neurodevelopmental presentations including intellectual disability, seizures, malformations of cortical development (abnormal brain MRI findings such as pachygyria, heterotopias, enlarged ventricles, hypoplasia of CC, brain stem, cerebellum), autism, and less frequently, neuromuscular phenotypes." [https://clinicalgenome.org/affiliation/40006/, https://www.ncbi.nlm.nih.gov/books/NBK1116/?term=dync1h1, PMID:32656949, PMID:32788638, PMID:33991169]
```

Comment:
Since you need to add the term tracker item, I'll make this minor suggestion to start this definition with "A spectrum of diseases...

also disease should be disease**s**

Reply:
The source "https://www.ncbi.nlm.nih.gov/books/NBK1116/?term=dync1h1" should be replaced with PMID:38513047

(they linked to the search page, but the actual page is: https://www.ncbi.nlm.nih.gov/books/NBK601997/ and there are PMIDs associated with gene reviews pages, so we want to use that instead of the URL. Thanks!

Reply:
Comments addressed